js/interface.js

```diff
@@ -63,54 +65,54 @@ Fliplet().then(function () {
   checkMultipleFieldToggles();
 
   if (data.headers && data.headers['Reply-To']) {
-   showGroup($('[name="replyTo"]').val(data.headers['Reply-To']));
-   $('[data-display="replyTo"]').hide();
+    showGroup($('[name="replyTo"]').val(data.headers['Reply-To']));
+    $('[data-display="replyTo"]').hide();
   }
 
-  $('[data-display]').click(function (event) {
+  $('[data-display]').click(function(event) {
     event.preventDefault();
     showGroup($('input[name="' + $(this).data('display') + '"]'));
     $(this).remove();
     checkMultipleFieldToggles();
     Fliplet.Widget.autosize();
   });
 
-  $('form').submit(function (event) {
+  $('form').submit(function(event) {
     event.preventDefault();
 
     data.headers = data.headers || {};
     data.to = [];
     data.subject = $('[name="subject"]').val();
 
-    multipleFields.forEach(function (name) {
+    multipleFields.forEach(function(name) {
       var value = $('[name="' + name + '"]').val();
+
       if (!value) {
         return;
       }
 
-      multiple(name, value).forEach(function (email) {
+      multiple(name, value).forEach(function(email) {
         if (email) {
           data.to.push(email);
         }
       });
     });
 
     var replyTo = $('[name="replyTo"]').val();
-    if (replyTo) {
```

Comment:
This `if` check shouldn't be removed. We're only setting a reply-to email address if it's provided. The system already has a default to use `no-reply@fliplet.com`. There's no need to save the `no-reply@fliplet.com` value.

Reply:
Ok. I'll fix it